### ISSUE TITLE
Broken link

### ISSUE THREAD

**Issue by flanaras** [kaigai/ssd2gpu#10]
Hello,

The pointer to the new repository gives a 404 error.
https://github.com/pg-strom/nvme-strom

Could you please give me some direction for the newer version of this repository?

Thanks

**Comment by kaigai** [comment]
You can get the driver from https://heterodb.github.io/swdc/

Also see http://heterodb.github.io/pg-strom/install/

**Comment by flanaras** [comment]
Thanks for your reply.

As far I as I can see, the source code is not included in the packages. Is that correct?

**Comment by kaigai** [comment]
Yes, because of our business reason, we stop to distribute its source code. sorry.
